### ISSUE TITLE
Deactivating a mapping should show the original segmentation, even when loading the mapping

### ISSUE THREAD

**Issue by jstriebel** [scalableminds/webknossos#4878]
I activated a large agglomerate-file (22GB) as a mapping in webknossos. Since it did not load immediately (the segmentation vanished, no loading indicator showed up), I tried to deactivate the mapping, which did not show the segmentation again. It would be great to show the segmentation then, and possibly also while still loading the mapping, and/or showing a loading indication.

While activating and deactivating the mapping, the following frontend-errors showed up in the console:
```
bucket.js:364 Uncaught (in promise) Error: Unexpected state: UNREQUESTED
    at T.unexpectedState (bucket.js:364)
    at T.markAsFailed (bucket.js:324)
    at pullBatch (pullqueue.js:112)
unexpectedState @ bucket.js:364
markAsFailed @ bucket.js:324
pullBatch @ pullqueue.js:112
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
(anonymous) @ layer_rendering_manager.js:251
Promise.then (async)
updateDataTextures @ layer_rendering_manager.js:225
update @ scene_controller.js:356
onPlaneViewRender @ plane_controller.js:370
h @ backbone-events-standalone.js:225
trigger @ backbone-events-standalone.js:168
renderFunction @ plane_view.js:124
animate @ plane_view.js:103
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
3bucket.js:364 Uncaught (in promise) Error: Unexpected state: UNREQUESTED
    at T.unexpectedState (bucket.js:364)
    at T.markAsFailed (bucket.js:324)
    at pullBatch (pullqueue.js:112)
unexpectedState @ bucket.js:364
markAsFailed @ bucket.js:324
pullBatch @ pullqueue.js:112
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
pullBatch @ pullqueue.js:124
async function (async)
pullBatch @ pullqueue.js:88
pull @ pullqueue.js:71
(anonymous) @ layer_rendering_manager.js:251
Promise.then (async)
updateDataTextures @ layer_rendering_manager.js:225
update @ scene_controller.js:356
onPlaneViewRender @ plane_controller.js:370
h @ backbone-events-standalone.js:225
trigger @ backbone-events-standalone.js:168
renderFunction @ plane_view.js:124
animate @ plane_view.js:103
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
requestAnimationFrame (async)
animate @ plane_view.js:105
(anonymous) @ plane_view.js:105
bucket.js:364 Uncaught (in promise) Error: Unexpected state: UNREQUESTED
    at T.unexpectedState (bucket.js:364)
    at T.markAsFailed (bucket.js:324)
    at pullBatch (pullqueue.js:112)
```

And @fm3 noticed this backend error during the same time:
```
@7hefg80b6 - Internal server error, for (POST) [/data/datasets/<orga>/<dataset>/layers/segmentation/data?token=<token> ->
ArrayIndexOutOfBoundsException: null
```

**Comment by fm3** [comment]
@youri-k was this fixed by #4904 ? :)

**Comment by youri-k** [comment]
Yes :)
